api/internal/target/kusttarget.go

```diff
@@ -151,20 +153,20 @@ func (kt *KustTarget) addHashesToNames(
 // holding customized resources and the data/rules used
 // to do so.  The name back references and vars are
 // not yet fixed.
-func (kt *KustTarget) AccumulateTarget() (
+func (kt *KustTarget) AccumulateTarget(origin *resource.Origin) (
```

Comment:
This is an important entry point, and it's argument should be explained.

Does the method change it / take ownership of it (presumably yes, since a ptr is sent). 
Is the outside owner expected to consult it?
What makes it change?

Reply:
added a comment explanation